CMakeLists.txt

```diff
@@ -29,6 +29,7 @@ CMAKE_DEPENDENT_OPTION(OPENDHT_HTTP "Build embedded http(s) client" OFF "NOT OPE
 option (OPENDHT_PEER_DISCOVERY "Enable multicast peer discovery" ON)
 option (OPENDHT_INDEX "Build DHT indexation feature" OFF)
 option (OPENDHT_TESTS "Add unit tests executable" OFF)
+option (OPENDHT_DISABLE_NETWORK_TESTS "Disable unit tests that require network access" OFF)
```

Comment:
option (OPENDHT_TESTS_NETWORK "Add unit tests that require network access" ON)

Reply:
Sure :) done.